README.md

```diff
@@ -14,87 +14,87 @@ Supports Node.js v4, v6 and v7 on Linux, MacOS, Windows and AIX.
 ## Usage
 
 ```bash
-npm install nodereport
-node -r nodereport app.js
+npm install node-report
+node -r node-report app.js
 ```
-A NodeReport will be triggered automatically on unhandled exceptions and fatal
+A node-report will be triggered automatically on unhandled exceptions and fatal
 error events (for example out of memory errors), and can also be triggered
 by sending a USR2 signal to a Node.js process (Linux/MacOS only).
 
-A NodeReport can also be triggered via an API call from a JavaScript
+A node-report can also be triggered via an API call from a JavaScript
 application.
 
 ```js
-var nodereport = require('nodereport');
-nodereport.triggerReport();
+var node-report = require('node-report');
```

Comment:
this is syntactically invalid javascript, you need to be more discerning in your search and replace. revert it to the original.